app/src/test/java/io/intrepid/skeleton/rules/MockServerRule.java

```diff
@@ -0,0 +1,52 @@
+package io.intrepid.skeleton.rules;
+
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+import io.intrepid.skeleton.testutils.TestFileUtils;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+
+/**
+ * A test rule that sets up and starts a mock server before the test, and then shut it down after the
+ * test is finished
+ */
+public class MockServerRule implements TestRule {
```

Comment:
Does this want to live in a separate library of test utilities?  I feel like some people may want to use the `MockRetrofit` style instead of the `MockWebServer` approach.